### ISSUE TITLE
Up arrow does not work (in tig, nano...)

### ISSUE THREAD

**Issue by matthiasbeyer** [zellij-org/zellij#1686]
**Basic information**

`zellij --version`: 0.31.2
`stty size`: 75 317
`uname -av` or `ver`(Windows): Linux REDACTED 5.15.59 #1-NixOS SMP Wed Aug 3 10:03:56 UTC 2022 x86_64 GNU/Linux

```
$ tig --version
tig version 2.5.6
ncursesw version 6.3.20220507
readline version 6.3
```

**Further information**

The UP arrow does not work properly in `tig` inside zellij. I confirmed that it works outside of zellij. 

I also tested `nano`. When pressing UP the cursor moves RIGHT, so it seems that some mapping internally went wrong.

I did not alter the default configuration besides removing the "Quit" mapping.

**Comment by imsnif** [comment]
Hey, this is fixed in 0.31.3 - it was a regression we hotfixed ASAP.

**Comment by matthiasbeyer** [comment]
Awesome! :tada:
